cmd/goal/clerk.go

```diff
@@ -152,6 +153,7 @@ func init() {
 	simulateCmd.Flags().StringVar(&requestFilename, "request", "", "Simulate request object to run. Mutually exclusive with --txfile")
 	simulateCmd.Flags().StringVar(&requestOutFilename, "request-only-out", "", "Filename for writing simulate request object. If provided, the command will only write the request object and exit. No simulation will happen")
 	simulateCmd.Flags().StringVarP(&outFilename, "result-out", "o", "", "Filename for writing simulation result")
+	simulateCmd.Flags().BoolVar(&signaturesOptional, "signatures-optional", false, "Allow transactions without signatures to be evaluated as if they had correct signatures")
 	simulateCmd.Flags().BoolVar(&liftLogLimits, "lift-log-limits", false, "Lift the limits on log opcode during simulation")
```

Comment:
sure, I think prefixing with `allow-*` or `relax-*` both sounds fine with me for consistent cmdline arg name.

We don't want to change internal var names, right?

Reply:
I'm open to `allow-empty-signatures` -- this was almost what I chose to call it anyway.

@jannotti are you suggesting changes for goal and for the REST API definitions? Right now they are the same

Reply:
I guess I'm suggesting it as widely as you'd like to make the changes.  I can understand not changing the equivalent underlying variables, but also, I guess it would be nice. Same for REST.

Reply:
Renamed in 1e1fdbdd7 to `allow-empty-signatures` and `allow-more-logging` everywhere